core/benchmarks/MapStatusesConvertBenchmark-jdk11-results.txt

```diff
@@ -2,12 +2,12 @@
 MapStatuses Convert Benchmark
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 11.0.16+8-LTS on Linux 5.15.0-1019-azure
+OpenJDK 64-Bit Server VM 11.0.16.1+1 on Linux 5.15.0-1022-azure
 Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 MapStatuses Convert:                      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Num Maps: 50000 Fetch partitions:500               1269           1276           8          0.0  1268666001.0       1.0X
-Num Maps: 50000 Fetch partitions:1000              2672           2695          39          0.0  2671542753.0       0.5X
-Num Maps: 50000 Fetch partitions:1500              4034           4069          50          0.0  4033696987.0       0.3X
+Num Maps: 50000 Fetch partitions:500               1227           1262          47          0.0  1226744907.0       1.0X
```

Comment:
Just out of curiosity, seems like the `Stdev` slow-dowm almost 6x, it is acceptable performance degradation?

Reply:
`Stdev` refers to the standard deviation estimated based on the sample. The standard deviation reflects the dispersion of the value relative to the mean. 

From `Per Row(ns)`, although the cpu frequency of the tester is slower than the base, the performance has not changed significantly